src/main/java/address/shortcuts/Bindings.java

```diff
@@ -0,0 +1,161 @@
+package address.shortcuts;
+
+import address.events.*;
+import javafx.scene.input.KeyCode;
+import javafx.scene.input.KeyCodeCombination;
+import javafx.scene.input.KeyCombination;
+import javafx.scene.input.KeyEvent;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+/**
+ * Contains the mapping of shortcuts and the corresponding event to raise
+ */
+public class Bindings {
+
+    /**
+     * List of accelerators used.
+     * They are here for the purpose of record keeping. Handled automatically by JavaFX.
+     */
+    private List<Accelerator> accelerators = new ArrayList<>();
+
+    /**
+     * List of Global hotkeys.
+     */
+    private List<GlobalHotkey> hotkeys = new ArrayList<>();
+
+    /**
+     * List of keyboard shortcuts.
+     */
+    private List<Shortcut> shortcuts = new ArrayList<>();
+
+
+    /* Global hotkeys in alphabetical order of name */
+    public List<GlobalHotkey> HOTKEY_APP_MINIMIZE = new ArrayList<>();
+    public List<GlobalHotkey> HOTKEY_APP_MAXIMIZE = new ArrayList<>();
+
+    /* Accelerators in alphabetical order of name */
+    public Accelerator ACCELERATOR_FILE_NEW;
+    public Accelerator ACCELERATOR_FILE_OPEN;
+    public Accelerator ACCELERATOR_FILE_SAVE;
+    public Accelerator ACCELERATOR_FILE_SAVE_AS;
+    public Accelerator ACCELERATOR_PERSON_DELETE;
+    public Accelerator ACCELERATOR_PERSON_EDIT;
+
+    /* Shortcuts in alphabetical order of name */
+    public Shortcut SHORTCUT_LIST_ENTER;
+
+    public Bindings(){
+        init();
+    }
+
+    private void init(){
+
+        /*====== bindings A-Z keys (in alphabetical order of main key =====================*/
+
+        ACCELERATOR_PERSON_DELETE = setAccelerator(KeyCode.D);
+
+        ACCELERATOR_PERSON_EDIT = setAccelerator(KeyCode.E);
+
+        ACCELERATOR_FILE_NEW = setAccelerator(KeyCode.N, KeyCombination.CONTROL_DOWN);
+
+        ACCELERATOR_FILE_OPEN = setAccelerator(KeyCode.O, KeyCombination.CONTROL_DOWN);
+
+        ACCELERATOR_FILE_SAVE = setAccelerator(KeyCode.S, KeyCombination.CONTROL_DOWN);
+
+        ACCELERATOR_FILE_SAVE_AS = setAccelerator(KeyCode.S, KeyCombination.CONTROL_DOWN, KeyCombination.ALT_DOWN);
```

Comment:
For all `CONTROL_DOWN`, we can try `SHORTCUT_DOWN` which *might* be correctly mapped to `Cmd` on Mac OS X and `Ctrl` on Windows (even if run on Macbook), as compared to `META_DOWN`. Otherwise, we can keep it to `CONTROL_DOWN`.